README.md

```diff
@@ -1,4 +1,37 @@
-# CommandBusLauncherBundle
+[![Build Status](https://travis-ci.org/ClearcodeHQ/command-bus-console.svg?branch=master)](https://travis-ci.org/ClearcodeHQ/command-bus-console)
+[![Coverage Status](https://coveralls.io/repos/ClearcodeHQ/command-bus-console/badge.svg?branch=master&service=github)](https://coveralls.io/github/ClearcodeHQ/command-bus-console?branch=master)
 
-[![Build Status](https://travis-ci.org/ClearcodeHQ/CommandBusLauncherBundle.svg?branch=master)](https://travis-ci.org/ClearcodeHQ/CommandBusLauncherBundle)
-[![Coverage Status](https://coveralls.io/repos/ClearcodeHQ/CommandBusLauncherBundle/badge.svg?branch=master&service=github)](https://coveralls.io/github/ClearcodeHQ/CommandBusLauncherBundle?branch=master)
+# Command Bus Console
+
+Command Bus Console is a package exposing your command bus functionality to the CLI.
+
+# Installation
+
+```
+$ composer require clearcode/command-bus-console
+```
+
+Enable `CommandBusConsole\Bundle\CommandBusConsoleBundle\CommandBusConsoleBundle` in the kernel of your Symfony application.
+
+```php
+    <?php
+    // app/AppKernel.php
+
+    public function registerBundles()
+    {
+        $bundles = array(
+            // ...
+            new CommandBusConsole\Bundle\CommandBusConsoleBundle\CommandBusConsoleBundle(),
```

Comment:
shouldn't it be `new Clearcode\CommandBusConsole\Bundle\CommandBusConsoleBundle()` ?